docs/docs/shared/deploy.mdx

```diff
@@ -16,24 +16,24 @@ Manual deployment can be a cause of bugs and mismatches in this regard so we don
 To deploy manually you can use the following command.
 
 ```bash
-bonsol deploy -m ./path-to-your-manifest.json -t {s3|shadowdrive|manual}
+bonsol deploy -m ./path-to-your-manifest.json {s3|shadowdrive|url}
```

Comment:
I think it would be good to leave the argument indicating the type, or to make s3 shadowdrive and url into a flag.

Reply:
Actually, I would need to update this, that's my bad. It should be:
`bonsol deploy {s3|shadowdrive|url} -m ./path-to-your-manifest.json`

Reply:
Updated with what that would look like